docs/source/upgrade-guide/index.md

```diff
@@ -65,6 +91,10 @@ This setting is no longer necessary.
 
 ### New dependencies synchronizer
 
+```{deprecated} Volto 18.0.0-alpha.43
+This step is only valid for projects that choose to continue using `yarn`-based generator and they haven't updated to Cookieplone `pnpm`-based setups.
+```
```

Comment:
Should we link back to upgrade-18-cookieplone-label from these notices?

Reply:
Agreed. I added a suggestion in the two admonitions.

Reply:
Done.